lib/test.d.ts

```
@@ -0,0 +1,234 @@
+    ): void;
+    throws(
+        this: Test,
+        fn: () => void,
+        msg: string,
+        extra?: AssertOptions,
+    ): void;
+
+    doesNotThrow(
+        this: Test,
+        fn: () => void,
+        exceptionExpected: RegExp | Function | undefined,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)